### ISSUE TITLE
Blockly timer reschedule only triggers once

### ISSUE THREAD

**Issue by 1Joe1** [openhab/openhab-webui#2072]
## The problem

Since updating to Openhab 4 all rules with a rescheduling timer only trigger the reschedule command once. 
There are no errors in the log file.

## Expected behavior

It should be possible to reschedule the timer more than once. Before updating to OpenHAB 4 my rules with rescheduling timers worked just fine.

## Steps to reproduce

Here is a an example 

`var n;

n = 1;
if (cache.private.exists('MyTimer') === false || cache.private.get('MyTimer').hasTerminated()) {
  cache.private.put('MyTimer', actions.ScriptExecution.createTimer('MyTimer', time.ZonedDateTime.now().plusSeconds(5), function () {
    if (n < 10) {
      items.getItem('Echo_Ankuendigung_Alle_Raeum').sendCommand(n);
      n = (typeof n === 'number' ? n : 0) + 1;
      if (cache.private.exists('MyTimer')) { cache.private.get('MyTimer').reschedule(time.ZonedDateTime.now().plusSeconds(5)); };
    }
    cache.private.remove('MyTimer');
  }));
};`

## Your environment

```yaml
runtimeInfo:
  version: 4.0.3
  buildString: Release Build
locale: de-DE
systemInfo:
  configFolder: /etc/openhab
  userdataFolder: /var/lib/openhab
  logFolder: /var/log/openhab
  javaVersion: 17.0.8.1
  javaVendor: Private Build
  osName: Linux
  osVersion: 5.4.0-162-generic
  osArchitecture: amd64
  availableProcessors: 4
  freeMemory: 127343616
  totalMemory: 486539264
  startLevel: 70
addons:
  - automation-jsscripting
  - binding-amazonechocontrol
  - binding-astro
  - binding-avmfritz
  - binding-mail
  - binding-mqtt
  - binding-network
  - binding-openweathermap
  - binding-samsungtv
  - binding-shelly
  - binding-systeminfo
  - misc-openhabcloud
  - persistence-rrd4j
  - transformation-jsonpath
  - ui-basic
  - ui-habpanel
clientInfo:
  device:
    ios: false
    android: true
    androidChrome: true
    desktop: false
    iphone: false
    ipod: false
    ipad: false
    edge: false
    ie: false
    firefox: false
    macos: false
    windows: false
    cordova: false
    phonegap: false
    electron: false
    nwjs: false
    os: android
    osVersion: "10"
    webView: false
    webview: false
    standalone: false
    pixelRatio: 2.75
    prefersColorScheme: dark
  isSecureContext: true
  locationbarVisible: true
  menubarVisible: true
  navigator:
    cookieEnabled: true
    deviceMemory: 4
    hardwareConcurrency: 8
    language: de-DE
    languages:
      - de-DE
      - en-US
    onLine: true
    platform: Linux armv81
  screen:
    width: 393
    height: 851
    colorDepth: 24
  support:
    touch: true
    pointerEvents: true
    observer: true
    passiveListener: true
    gestures: false
    intersectionObserver: true
  themeOptions:
    dark: dark
    filled: true
    pageTransitionAnimation: default
    bars: light
    homeNavbar: default
    homeBackground: default
    expandableCardAnimation: default
  userAgent: Mozilla/5.0 (Linux; Android 10; Redmi Note 7 Build/QKQ1.190910.002;
    wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/116.0.0.0
    Mobile Safari/537.36
timestamp: 2023-09-16T20:28:36.934Z

```

## Browser console

```txt

```

## Additional information

The issue might be related to this older issue.[[BLOCKY] reschedule block triggers only once #1214](https://github.com/openhab/openhab-webui/issues/1214#issue-1065276574) 
Everything seems to be very similar.

**Comment by florian-h05** [comment]
@stefan-hoehn Fixed with #2023?

**Comment by stefan-hoehn** [comment]
Yes, that is exactly the problem that was fixed with #2032

**Comment by florian-h05** [comment]
Great, let’s close this then.

@1Joe1 The fix is already backported, it will be available with the next patch release (4.0.4).
